utils/dwdparse/dwdparse_test.go

```diff
@@ -238,174 +241,187 @@ const (
-	{"#DW jobdw type=xfs     capacity=100TB name=!!21//\\                                     ", allow, invalidDW},
-	{"#DW jobdw                                                                               ", allow, invalidDW},
-
-	{"#DW container name=mycontainer spec=some-repo-name job_storage={stor1,} supervisor=rabbit", deny, invalidDW},
-	{"#DW container name=mycontainer spec=some-repo-name job_storage={,stor1} supervisor=rabbit", deny, invalidDW},
-	{"#DW container name=mycontainer ", deny, invalidDW},
+	{[]string{"#DW jobdw type=raw    capacity=100GB name=prettyGoodName  "}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=xfs    capacity=100GB name=pretty_GoodName "}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=gfs2   capacity=100GB name=pretty_GoodName "}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=lustre capacity=100GB name=prettyGood-Name "}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=raw    capacity=100TB name=__prettyGoodName"}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=xfs    capacity=100TB name=-prettyGoodName-"}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=lustre capacity=100TB name=0prettyGoodName1"}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=raw    capacity=100GB name=prettierGoodName"}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=xfs    capacity=100GB name=prettierGoodName"}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=lustre capacity=100GB name=prettierGoodName"}, deny, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=lustre capacity=100GB name=prettierGoodName"}, deny, validDWOrAllowUnknownCommand},
+
+	{[]string{"#DW jobdw type=raw    capacity=100GB name=prettyGoodName  "}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=xfs    capacity=100GB name=prettyGoodName  "}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=lustre capacity=100GB name=prettyGoodName  "}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=raw    capacity=100TB name=prettyGoodName  "}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=xfs    capacity=100TB name=prettyGoodName  "}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=lustre capacity=100TB name=prettyGoodName  "}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=raw    capacity=100GB name=prettierGoodName"}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=xfs    capacity=100GB name=prettierGoodName"}, allow, validDWOrAllowUnknownCommand},
+	{[]string{"#DW jobdw type=lustre capacity=100GB name=prettierGoodName"}, allow, validDWOrAllowUnknownCommand},
+
+	{[]string{"#DW jobdw type=raw    capacity=100GB name=uniqueName1 ", "#DW jobdw type=raw    capacity=100GB name=uniqueName2 "}, allow, validDWOrAllowUnknownCommand},
```

Comment:
Can you split these lines so they stand out as multi-DW to the eye?  Will go fmt allow that?